packages/terra-icon/CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ Changelog
 Unreleased
 ----------
 
+### Changed
+* Added Reply All Icon
```

Comment:
Why is this in Changed and the others in Added?

Reply:
Because the icon existed and was modified.